### PR TITLE
add auto disconnect and a LOT of misc changes to music commands

### PR DESCRIPTION
**Pull request by IllusionMan1212**
- fixed skip command
- formatted music file
- bot disconnects automatically after 2 minutes
- other things ???? i dont remember lol

### CODE REVIEW

File: Vita3KBot/Bot.cs [hunk 1]
```diff
@@ -5,7 +5,6 @@
 using Discord;
 using Discord.WebSocket;
 using Victoria;
-using Victoria.EventArgs;
```
Comment (freza1) [inline Vita3KBot/Bot.cs:8]: why take it out?

File: Vita3KBot/QueueManager.cs [hunk 2]
```diff
@@ -39,5 +39,9 @@ public static List<LavaTrack> PlayList(this ulong guildId)
             Queue.TryAdd(guildId, new Queue<LavaTrack>());
             return Queue[guildId].ToList();
         }
+
+        public static int GetQueueCount(this ulong guildId) {
+            return Queue[guildId].Count;
+        }
```
Comment (freza1) [inline Vita3KBot/QueueManager.cs:45]: what is this supposed to return? if it's the queue count, didn't i add that?

File: Vita3KBot/Bot.cs (hunk 1 above)
Comment (IllusionMan1212) [reply Vita3KBot/Bot.cs:8]: It's not used, vsc and vs dim the unused directives so it's easier to tell

File: Vita3KBot/QueueManager.cs (hunk 2 above)
Comment (IllusionMan1212) [reply Vita3KBot/QueueManager.cs:45]: Afaik there's no way to get the queue count unless we do this or make the queue property public

**Pull request merged**